go/host/enclave/service.go

```diff
@@ -122,7 +122,15 @@ func (e *Service) GetEnclaveClients() []common.Enclave {
 	return clients
 }
 
-func (e *Service) EvictEnclave(enclaveID *common.EnclaveID) {
+func (e *Service) NotifyUnavailable(enclaveID *common.EnclaveID) {
+	if len(e.enclaveGuardians) <= 1 {
+		e.logger.Info("not running in HA mode, no need to evict enclave", log.EnclaveIDKey, enclaveID)
+		return
+	}
+	if *e.activeSequencerID != *enclaveID {
```

Comment:
any reason we can't compare the actual values here instead?

Reply:
I think this does compare the values (although I haven't fully internalised the quirks around this tbh lol). I had it as `e.activeSequencerID == enclaveID` below and Tudor fixed it, think it was comparing the pointers with that.